### ISSUE TITLE
Doesn't seem to work in Edge

### ISSUE THREAD

**Issue by adjenks** [matteobad/focus-within-polyfill#7]
It says: 

> focus-within-polyfill: focus-within not supported
> focus-within-polyfill: loaded.

But it doesn't seem to be working. Perhaps you need to do more than load it? Do you have to transform your style sheets or something?

**Comment by adjenks** [comment]
I'm using it for a customElement. Perhaps your polyfill doesn't pierce the shadow DOM.

**Comment by matteobad** [comment]
Hi @adjenks, thanks for the interest. Right now I tried to keep things small and simple. Two event listeners are attached to the document and when triggered the handler uses `document.activeElement` as a starting point, then a loop throw all the parents start and the `.focus-within` class is applied.

At the moment there is no support for shadow DOM, as soon as I have some spare time I'll try to do something about it. In the meantime, if you have some ideas you can even submit a PR.

**Comment by matteobad** [comment]
Since Shadow DOM is not natively supported in EDGE I don't think that is in the scope of this polyfill to support this. I've updated the README

**Comment by adjenks** [comment]
Ah, I was using a polyfill for shadow dom as well for Edge. focus-within already works in chrome and firefox, but not in Edge, and I wanted focus-within and customelements. I guess you can't polyfill the whole world...

**Comment by adjenks** [comment]
I want a polyfill for FireFox in Edge. 😂

**Comment by matteobad** [comment]
Hi @adjenks, sorry if I have dismissed this issue so soon. I think I've just found a way to pierce through the shadow dom both in 'open' and 'closed' mode. Basically a loop through `document.activeElement.shadowRoot` to find di active element even if inside a shadowRoot.

I've pushed the POC on develop with an updated example that uses the webcomponents polyfill for IE and Edge. I'll try to test and publish this feature in the next few days.
